config/routes.rb

```diff
@@ -3,4 +3,32 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  root 'application#welcome'
+
+  # Hotel
+  get '/hotels', to: 'hotels#index', as: 'hotels'
+  get '/hotels/:id', to: 'hotels#show', as: 'hotel'
+  get '/hotels/:id/edit', to: 'hotels#edit', as: 'edit_hotel'
+  patch '/hotels/update', to: 'hotels#update', as: 'update_hotel'
+  delete '/hotels/delete', to: 'hotels#destroy', as: 'delete_hotel'
+
+  # Room
+  get '/rooms', to: 'rooms#index', as: 'rooms'
+  get '/rooms/:id', to: 'rooms#show', as: 'room'
+  get '/rooms/:id/edit', to: 'rooms#edit', as: 'edit_room'
+  patch '/rooms/update', to: 'rooms#update', as: 'update_room'
+  delete '/rooms/delete', to: 'rooms#destroy', as: 'delete_room'
+
+  # Guest
+  get '/guests', to: 'guests#index', as: 'guests'
+  get '/guests/:id', to: 'guests#show', as: 'guest'
+  get '/guests/:id/edit', to: 'guests#edit', as: 'edit_guest'
+  patch '/guests/:id', to: 'guests#update', as: 'update_guest'
+  post '/guests/:id/add_room', to: 'guests#add_room', as: 'add_room_guest'
```

Comment:
A better route might be `post "/guests/:id/rooms`, which would go to the `stays#create` controller & action.